packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -3658,6 +3797,48 @@ module.exports = function (log, config, bounces, statsd) {
     )}`;
   };
 
+  Mailer.prototype._generateCmsLinks = async function (
+    planId,
+    acceptLanguage,
+    templateName
+  ) {
+    if (
+      !(planId && acceptLanguage && Container.has(ProductConfigurationManager))
+    ) {
+      return {};
+    }
+
+    const productConfigurationManager = Container.get(
+      ProductConfigurationManager
+    );
+
+    const purchaseDetails =
+      await productConfigurationManager.getPurchaseWithDetailsOfferingContentByPlanIds(
+        [planId],
+        acceptLanguage
+      );
+    const transformedPurchaseDetails =
+      purchaseDetails.transformedPurchaseWithCommonContentForPlanId(planId);
+    transformedPurchaseDetails.offering.commonContent.privacyNoticeDownloadUrl;
+
+    return {
+      subscriptionPrivacyUrl: this._generateUTMLink(
+        transformedPurchaseDetails.offering.commonContent
```

Comment:
fix: update to use localization

To get the localized string, this needs to be updated to something like the code below.

```
transformedPurchaseDetails.offering.commonContent.localizations[0].privacyNoticeDownloadUrl
```